schema/artwork/index.js

```diff
@@ -39,6 +39,10 @@ import {
   GraphQLInt,
 } from 'graphql';
 
+const is_inquireable = ({ inquireable, acquireable }) => {
+  return (inquireable && !acquireable);
+};
```

Comment:
I know that this logic predates this PR, but I’m slightly confused with the naming of things here. There’s a `inquireable` property that comes from Gravity, but MP’s version of `is_inquireable` _also_ takes into account an extra property `acquireable`. What gives?

Off-topic shouldn’t `acquireable` be spelled `acquirable`?